utils/download_test_db.py

```diff
@@ -121,31 +121,88 @@ def download_dataset(dataset):
             # Just a newline.
             print()
 
+def download_caffe2_models ():
+    models = ['densenet121',
+              'inception_v1', 
+              'inception_v2',
+              'lenet_mnist',
+              'resnet50',
+              'shufflenet',
+              'squeezenet',
+              'vgg19', 
+              'zfnet512',
+              'bvlc_alexnet',
+              'en2gr',
+              'quant_resnet50']
+    for modelname in models:
+        for filename in ['predict_net.pbtxt', 'predict_net.pb', 'init_net.pb']:
+            os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/{}/{} -P  {}'.format(modelname, filename, modelname))
+    
+    for filename in ['dst_dictionary.txt', 'src_dictionary.txt']:
+        os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/en2gr/{} -P en2gr'.format(filename))
+    return
+
+def download_onnx_models ():
+    for modelname in ['resnet50', 'vgg19', 'squeezenet', 'zfnet512', 'densenet121', 'shufflenet']:
+        os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_6/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['inception_v1', 'inception_v2', 'bvlc_alexnet']:
+        os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_8/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['lenet_mnist']:
+        os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['googlenet_v1_slim', 'googlenet_v4_slim', 'resnet50_tf']:
+        os.system ('mkdir {}'.format(modelname))    
+        os.system ('wget -nc -P {} http://fb-glow-assets.s3.amazonaws.com/models/{}.onnx'.format(modelname, modelname))
+
+    os.system ('wget -nc https://onnxzoo.blob.core.windows.net/models/opset_8/emotion_ferplus/emotion_ferplus.tar.gz')
+    os.system ('tar -xzvf emotion_ferplus.tar.gz emotion_ferplus/model.onnx')
+
+    os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_8/bvlc_reference_rcnn_ilsvrc13.tar.gz')
+    os.system ('tar -xzvf bvlc_reference_rcnn_ilsvrc13.tar.gz bvlc_reference_rcnn_ilsvrc13/model.onnx')
+    return
 
 def parse():
     parser = argparse.ArgumentParser(description='Download datasets for Glow')
     parser.add_argument('-d', '--datasets', nargs='+', choices=DATASET_NAMES)
-    parser.add_argument('-a', '--all', action='store_true')
+    parser.add_argument('-ad', '--allDatasets', action='store_true')
+    parser.add_argument('-c', '--caffe2Models', action='store_true')
+    parser.add_argument('-o', '--onnxModels', action='store_true')
     options = parser.parse_args()
 
-    if options.all:
+    if options.allDatasets:
         datasets = DATASET_NAMES
```

Comment:
If we specify all datasets, shouldn't we also set caffe2 and onnx models to also download here?

Reply:
i thought datasets don't include models as datasets are inputs to the model. Maybe we can use flags like -ac, -ao to download all caffe2/onnx models

Reply:
Ah yeah, you're right. Perhaps we should rename this script, and also update the description for ArgParse too.

Reply:
I like the idea of renaming the script to reflect what it does now

Reply:
Not: `-ac` usually means both `-a` and `-c`, right? I haven't really seen two lettter short forms for flags.

Reply:
That's right. So i guess we just go with --all-datasets and --all-caffe2-modes, --all-onnx-models? 
But it seems too long for ppl to type every time, lol

Reply:
i guess i can use -D, -C, -O to specify all datasets and all models.

Reply:
changed the file name to download_datasets_and_models.py